library/cartridge_connect_to_membership.py

```diff
@@ -32,6 +32,9 @@ def connect_to_membership(params):
     changed = False
 
     for instance_name, instance_vars in hostvars.items():
+        if 'role_vars' in instance_vars:
```

Comment:
What if `role_vars` is not in `instance_vars`?

Reply:
Why not `instance_vars = hostvars[instance_name]['role_vars']` (as below)?

Reply:
Because some instances are not in play hosts, so, they don't have `role_vars`